hooks/tk-multi-shotgunpanel/tk-nuke_actions.py

```diff
@@ -292,7 +293,7 @@ def _find_sequence_range(self, path, sg_data):
         :returns: None if no range could be determined, otherwise (min, max)
         """
         # find a template that matches the path:
-
+        '''
```

Comment:
Please add a comment as to why this block is being commented out with associated ticket number.

Reply:
I will remove the code block